src/vl-proza-message.js

```diff
@@ -71,8 +71,11 @@ export class VlProzaMessage extends vlElement(HTMLElement) {
   }
 
   connectedCallback() {
-    this.appendChild(this.__createWysiwygElement());
-    this.__processToegelatenOperaties();
```

Comment:
Was er geen mogelijkheid om dit naar de constructor te verplaatsen?

Reply:
@coemans dacht dat we dan tegen andere problemen zouden aanlopen, dus niet onderzocht moet ik toegeven.